pkg/services/sqlstore/migrator/sqlite_dialect.go

```diff
@@ -85,6 +86,39 @@ func (db *Sqlite3) CleanDB() error {
 	return nil
 }
 
+// TruncateDBTables deletes all data from all the tables and resets the sequences
+// A special case is the dashboard_acl table where we keep the default permissions
+func (db *Sqlite3) TruncateDBTables() error {
+	tables, err := db.engine.DBMetas()
+	if err != nil {
+		return err
+	}
+
+	sess := db.engine.NewSession()
+	defer sess.Close()
+
+	for _, table := range tables {
+		switch table.Name {
+		case "dashboard_acl":
+			// keep default dashboard permissions
+			if _, err := sess.Exec(fmt.Sprintf("DELETE FROM %q WHERE dashboard_id != -1 AND org_id != -1;", table.Name)); err != nil {
+				return errutil.Wrapf(err, "failed to truncate table %q", table.Name)
+			}
+			if _, err := sess.Exec("UPDATE sqlite_sequence SET seq = 2 WHERE name = '%s';", table.Name); err != nil {
```

Comment:
@kylebrandt @sakjur 
thank you for approving it!
However, I've introduced this change (together with @kylebrandt's suggestions) since the last time you checked so if you don't mind please have a look.
The change is basically for setting the `dashboard_acl` sequence to the correct value; note that no tests did complain before. I guess that's because SQLite is smart handling it but still it's good to fix it:
```
sqlite> UPDATE sqlite_sequence SET seq = 0 WHERE name = 'dashboard_acl';
sqlite> select id from dashboard_acl ;
1
2
3
4
sqlite> select * from sqlite_sequence where name = 'dashboard_acl';
dashboard_acl|0
sqlite> insert into dashboard_acl(org_id, dashboard_id, user_id, permission, created, updated) values (1, 138, 3, 4, '2020-10-15 10:24:48', '2020-10-15 10:24:48');
sqlite> select id from dashboard_acl ;
1
2
5
3
4
sqlite> select * from sqlite_sequence where name = 'dashboard_acl';
dashboard_acl|5
```